apps/postgres/create-hydra-db.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+psql --username docker <<-EOSQL
+	CREATE DATABASE hydra;
+EOSQL
```

Comment:
Is this something we could do in `apps/hydra/start.sh` instead?

Reply:
The reason I ask is that I'd like to keep things as isolated as possible. I think it should be possible if we install `psql` in the `hydra` container?